### PR TITLE
Create epsonprinter.markdown

### PR DESCRIPTION
**Pull request by ThaStealth**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23144

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/epsonworkforceprinter.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+date: 2019-04-09 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Sensor
+ha_release: 0.93
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/epsonworkforceprinter.markdown]: only for existing pages

File: source/_components/epsonworkforceprinter.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+     - clean   
+```
+
+{% configuration %}
+host:
+  description: The host name or address of the Epson printer
+  required: true
+  type: string
+monitored_conditions:
+  description: The cartridge colours to monitor. Valid values are: black,yellow,magenta,cyan and clean. Clean is the cleaning cartridge
```
Comment (klaasnicolaas) [inline source/_components/epsonworkforceprinter.markdown]:
```suggestion
  description: "The cartridge colours to monitor. Valid values are: `black`, `yellow`, `magenta`, `cyan` and `clean`. Clean is the cleaning cartridge."
```

File: source/_components/epsonworkforceprinter.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Assistant.
+
+To add Epson Workforce Printer to your installation,
+add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+   - platform: epsonworkforceprinter
+     host: 192.168.0.123
```
Comment (klaasnicolaas) [inline source/_components/epsonworkforceprinter.markdown]:
```suggestion
     host: IP_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/epsonworkforceprinter.markdown (hunk 1 above)
Comment (ThaStealth) [reply source/_components/epsonworkforceprinter.markdown]: Removed it

**Comment by balloob** [comment]
@ThaStealth you have closed this PR for the docs but the integration was merged. Please open a PR asap to fix this.

**Comment by ThaStealth** [comment]
@balloob 
Very strange, I do not know how I caused this.
Anyway, created a new PR at https://github.com/home-assistant/home-assistant.io/pull/9294

**Comment by balloob** [comment]
Thanks!

**Pull request closed without merging**